File: java/darts/src/main/java/Darts.java
import java.lang.Math;

class Darts {
    private int score;

    Darts(double x, double y) {
        double radius = Math.sqrt((x * x) + (y * y));
        if (radius > 10) {
            score = 0;
        } else if (radius > 5) {
            score = 1;
        } else if (radius > 1) {
            score = 5;
        } else {
            score = 10;
        }
    }

    int score() {
        return score;
    }

}

File: c/darts/src/darts.c
#include "darts.h"
#include <math.h>

int score(coordinate_t pos)
{
  float distance = sqrt(pos.x * pos.x + pos.y * pos.y);
  if (distance > 10) return 0;
  if (distance > 5) return 1;
  if (distance > 1) return 5;
  return 10;
}
